lib/car.rb

```diff
@@ -0,0 +1,19 @@
+class Car
+  attr_reader :make, :model, :monthly_payment, :loan_length, :color
+  def initialize(make_and_model, monthly_payment, loan_length)
+    make_model = make_and_model.split(" ")
+    @make = make_model.first
+    @model = make_model.last
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+    @color
+  end
```

Comment:
Good work using `first` and `last` after using the `.split` method on the `make_and_model` argument.